scripts/HFIR_4Circle_Reduction/absorption.py

```diff
@@ -1,23 +1,22 @@
 #pylint: disable=R0913,W0403,R0903,C0103
 from __future__ import (absolute_import, division, print_function)
-from six.moves import range
-import numpy
-import numpy.linalg
+import HFIR_4Circle_Reduction.fourcircle_utility as util4
 import math
 from mantid.api import AnalysisDataService
+import numpy
+import numpy.linalg
+from six.moves import range
 
-import HFIR_4Circle_Reduction.fourcircle_utility as util4
-
-# Do absorption correction
+# This module does absorption correction
 
 
 class Lattice(object):
-    """
-    A simple structure-styled class to hold lattice or lattice*
+    """A simple structure-styled class to hold lattice or lattice*
```

Comment:
This class should go away. We already have a `UnitCell` class in Mantid, or if you need the orientation as well, use `OrientedLattice`

Reply:
this will be resolved in #20937.